src/storekit.cs

```diff
@@ -132,34 +140,39 @@ interface SKMutablePayment {
 		[Availability (Deprecated = Platform.iOS_5_0, Message = "Use 'PaymentWithProduct (SKProduct)' after fetching the list of available products from 'SKProductRequest' instead.")]
 		SKMutablePayment PaymentWithProduct (string identifier);
 
+		[Watch (6, 2)]
 		[NullAllowed] // by default this property is null
 		[Export ("productIdentifier", ArgumentSemantic.Copy)][New]
 		string ProductIdentifier { get; set; }
 
+		[Watch (6, 2)]
 		[Export ("quantity")][New]
 		nint Quantity { get; set; }
 
+		[Watch (6, 2)]
 		[NullAllowed] // by default this property is null
 		[Export ("requestData", ArgumentSemantic.Copy)]
 		[Override]
 		NSData RequestData { get; set; }
 
-		[iOS (7,0), Mac (10, 9)]
+		[Watch (6, 2), iOS (7,0), Mac (10, 9)]
 		[NullAllowed] // by default this property is null
 		[Export ("applicationUsername", ArgumentSemantic.Copy)][New]
 		string ApplicationUsername { get; set; }
 
-		[iOS (8,3), Mac (10,14)]
+		[Watch (6, 2), iOS (8,3), Mac (10,14)]
```

Comment:
We should be consistent between [Watch (6, 2)] and [Watch (6,2)]. I know we are not very, but in this case you are using both in the same PR.